src/Entity/WebauthnCredential.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\Table;
+use Symfony\Component\Uid\AbstractUid;
+use Symfony\Component\Uid\Ulid;
+use Webauthn\PublicKeyCredentialSource as BasePublicKeyCredentialSource;
+use Webauthn\TrustPath\TrustPath;
+
+#[Table(name: "webauthn_credentials")]
+#[Entity(repositoryClass: WebauthnCredentialRepository::class)]
+class WebauthnCredential extends BasePublicKeyCredentialSource
+{
+    #[Id]
+    #[Column(type: Types::STRING, unique: true)]
```

Comment:
This `unique: true` is useless. A primary key is already a unique field.

Reply:
Correct. I removed it